libpod/options.go

```diff
@@ -1469,3 +1470,14 @@ func WithInfraContainerPorts(bindings []ocicni.PortMapping) PodCreateOption {
 		return nil
 	}
 }
+
+func WithHealthCheck(healthCheck *manifest.Schema2HealthConfig) CtrCreateOption {
```

Comment:
Needs comment

Reply:
Also - should we try and set this to the image's default in NewContainer?

Reply:
in the future it can be overriden in the command line, so id like to start as-is